src/cmake/externalpackages.cmake

```diff
@@ -198,25 +202,29 @@ checked_find_package (TBB 2017
                       PREFER_CONFIG)
 
 checked_find_package (DCMTK VERSION_MIN 3.6.1)  # For DICOM images
-checked_find_package (FFmpeg VERSION_MIN 3.0)
+checked_find_package (ffmpeg
+                      COMPONENTS avcodec avformat avutil swscale
+                      VERSION_MIN 3.0)
 checked_find_package (GIF
                       VERSION_MIN 4
                       RECOMMEND_MIN 5.0
                       RECOMMEND_MIN_REASON "for stability and thread safety")
 
 # For HEIF/HEIC/AVIF formats
-checked_find_package (Libheif VERSION_MIN 1.3
+checked_find_package (libheif VERSION_MIN 1.3
```

Comment:
Is the effect of this that we're relying on libheif's exported cmake config?

That might be fine, but if so, does that alter the minimum version of libheif that we support? (I.e., the fact that we have our own FindLibheif.cmake likely means that when we added libheif support, that package didn't have a reliable exported cmake config.)

Reply:
These come from Conan. So they are Conan generated configs.

Reply:
It looks like recent libheif exports cmake configs, and calls it libheif-config.cmake, so we should either change our FindLibheif.cmake module (which I believe I wrote before libheif exported cmake configs) to the lower case convention, or possibly eliminate it entirely if we are willing to bump our libheif minimum to whenever they started exporting cmake configs reliably.

Reply:
This kind of thing has happened many times over the years -- we needed a dependency that has no exported cmake config, so we made our own FindFoo module, and then years later, the dependency project starts exporting cmake config but uses a different capitalization than the one we originally guessed.

Reply:
Looks like we're back to ``Libheif`` and if I'm reading this correctly we need to "guess right" the first time?